src/js/fluxcontroller.js

```diff
@@ -755,6 +760,14 @@ define(function (require, exports, module) {
                 return actionPromise;
             })
             .tap(function () {
+                if (hideOverlays) {
+                    actionReceiver.dispatch(events.panel.END_CANVAS_UPDATE);
+                }
+
+                if (lockUI && !uiWasLocked) {
```

Comment:
Unrelated: I moved this into the previous `tap` since there wasn't any reason for an additional handling frame.